src/controller/tests/data_model/TestRead.cpp

```diff
@@ -1623,10 +1640,12 @@ void TestReadInteraction::TestReadHandler_KillOverQuotaSubscriptions(nlTestSuite
     {
         TestReadCallback callback;
         std::vector<std::unique_ptr<app::ReadClient>> outReadClient;
-        EstablishSubscriptions(apSuite, ctx.GetSessionBobToAlice(), 1,
-                               app::InteractionModelEngine::kMinSupportedPathsPerSubscription + 1, &callback, outReadClient);
+        EstablishReadOrSubscriptions(
```

Comment:
Splitted this into two parts.

Adds `Since ... if it exceeds minimas` before `SetForceHandlerQuota`

And the case description right before the case.